assignments/objects.js

```diff
@@ -19,46 +19,140 @@ let example = {
 
 // Write your intern objects here:
 
+let mitzy = {
+  id: 1,
+  name: "Mitzy",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+}
+let kennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function speak(){
+    phrase = "Hello my name is" + " " + this.name;
```

Comment:
Good opportunity for those template literals:
`return 'Hello my name is ${this.name}';`

So I used single quotes up there, but they should be backticks. I can't use backticks in here without github formatting them and thinking I'm trying to do something else, lol. Kind of annoying....